ios/RNMask.swift

```diff
@@ -10,8 +11,9 @@ import InputMask
 
 @objcMembers
 open class RNMask : NSObject {
-    public static func maskValue(text: String, format: String, autcomplete: Bool) -> String {
-        let mask : Mask = try! Mask.getOrCreate(withFormat: format)
+    public static func maskValue(text: String, format: String, autcomplete: Bool, options: NSDictionary) -> String {
+        let customNotations = (options["customNotations"] as? [[String:Any]])?.map { $0.toNotation() } ?? []
+        let mask : Mask = try! Mask.getOrCreate(withFormat: format, customNotations: customNotations)
```

Comment:
Similar to my comment on android, we should reuse this logic instead of copying it